benchmarker_utils.f90

```diff
@@ -40,6 +40,36 @@ subroutine assert_real_2d(a, b, test_name, rtol_opt)
 
   end subroutine assert_real_2d
 
+  subroutine assert_real(a, b, test_name, rtol_opt)
```

Comment:
can we do something like `print_time_stats` and make an interface for the assertions. `assert_real` could be the interface and we could make assert_real_1d for singular values, for example. Then in the main code we can just use `asset_real` for any comparison involving reals.

Reply:
Good idea, thanks! Done